src/safe-unsafe-meaning.md

```diff
@@ -41,12 +41,14 @@ The standard library has a number of unsafe functions, including:
 * All FFI (Foreign Function Interface) functions are `unsafe` to call because the
   other language can do arbitrary operations that the Rust compiler can't check.
 
-As of Rust 1.0 there are exactly two unsafe traits:
+As of Rust 1.29.2 there these stable unsafe traits (there are others, but they
```

Comment:
trying out this suggesting thing for the first time??

```suggestion
As of Rust 1.29.2 the standard library defines the following unsafe traits (there are others, but they
```

Reply:
As an aside the nomicon is not strictly bound to documenting stable things; indeed its early focus involved many unstable things.

Reply:
I noticed there are some unsafe things, but I thought it was more like out of necessity. Anyway, there's a bit too many of the unstable unsafe traits. I'm afraid the full list would look a bit scary.